channels/serializers.py

```diff
@@ -144,6 +145,13 @@ def get_url(self, instance):
         """Returns a url or null depending on if it's a self post"""
         return instance.url if not instance.is_self else None
 
+    def get_author_name(self, instance):
+        """get the authors name"""
+        user = self._get_user(instance)
+        if user and user.profile.name:
+            return user.profile.name
+        return "Someone"
```

Comment:
Could you change this to be "[deleted]" to preserve the original default value?

Reply:
sure